File: app/src/main/java/com/neome/java/api/meta/base/dto/StudioDtoMappingFieldMap.java
// neome.ai API - do not change
//

package com.neome.java.api.meta.base.dto;

import com.neome.java.api.meta.base.Types.EnumDefnEmptyFieldVariant;
import com.neome.java.api.meta.base.Types.EnumDefnInsertVariant;
import com.neome.java.api.meta.base.Types.EnumDefnRemoveVariant;
import com.neome.java.api.meta.base.Types.EnumDefnUpdateVariant;
import com.neome.java.api.meta.base.Types.MetaIdField;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoMappingFieldMap extends StudioDtoMappingFieldMapBase
{
  @Nullable
  public EnumDefnEmptyFieldVariant emptyFieldVariant;

  @Nullable
  public MetaIdField fromKey;

  @Nullable
  public EnumDefnInsertVariant insertVariant;

  @Nullable
  public EnumDefnRemoveVariant removeVariant;

  @Nullable
  public MetaIdField toKey;

  @Nullable
  public EnumDefnUpdateVariant updateVariant;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoMappingFieldMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Types.EnumDefnEmptyFieldVariant
import com.neome.api.meta.base.Types.EnumDefnInsertVariant
import com.neome.api.meta.base.Types.EnumDefnRemoveVariant
import com.neome.api.meta.base.Types.EnumDefnUpdateVariant
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.dto.StudioDtoMappingFieldMapBase

interface StudioDtoMappingFieldMap : StudioDtoMappingFieldMapBase
{
  val emptyFieldVariant: EnumDefnEmptyFieldVariant?
  val fromKey: MetaIdField?
  val insertVariant: EnumDefnInsertVariant?
  val removeVariant: EnumDefnRemoveVariant?
  val toKey: MetaIdField?
  val updateVariant: EnumDefnUpdateVariant?
}